### ISSUE TITLE
question

### ISSUE THREAD

**Issue by attiqa680** [BMIRDS/3dMRISegmentation#5]
Thankyou sir. 
One more question I want to ask that at training there is 1200 epochs and it will saves only best models but how we can find out which epoch is best. Secondly I want this  results shown in figure
![Capture1](https://user-images.githubusercontent.com/59041075/109913228-62a53200-7ccf-11eb-80ad-6474aecd4ca9.PNG)

**Comment by ntomita** [comment]
You can look at outputs from training script, which show the validation performance summarized at each epoch, and pick a model that has the lowest avg loss. To get those numbers I'd suggest you to take a look at https://github.com/deepmind/surface-distance
